### PR TITLE
Fix issue when no plugins are defined on the app

### PR DESCRIPTION
**Pull request by yannickglt**

### CODE REVIEW

File: lib/core/bootstrap.class.ts [hunk 1]
```diff
@@ -15,34 +15,39 @@ export function bootstrap(...BootstrapedApps: Array<IAppStatic>): Array<IApp> {
...
-      server.route(getRoutesWithConfigRecurs(BootstrapedApp))
-      server.start((err: any) => {
-        try {app.onStart()} catch(err) {}
+    const plugins: Array<IPlugin> = getPluginsRecurs(Reflect.getMetadata('hapiour:plugins', BootstrapedApp))
+    if (!_.isEmpty(plugins)) {
+      server.register(plugins, (err: any) => {
+        try {app.onRegister()} catch(err) {}
+        server.route(getRoutesWithConfigRecurs(BootstrapedApp))
+        server.start((err: any) => {
+          try {app.onStart()} catch(err) {}
```
Comment (heiso) [inline lib/core/bootstrap.class.ts]: so what if `plugins` is empty ? app will not start ?

Comment (yannickglt) [reply lib/core/bootstrap.class.ts]: you're right, I made a fixup.

Comment (heiso) [reply lib/core/bootstrap.class.ts]: thx 🥇

**Review by heiso — APPROVED** [review]
LGTM

**Comment by heiso** [comment]
If you are brave enough -> https://github.com/Boulangerie/hapiour-decorators/issues/8 :p
Else I will take care of it ASAP

**Pull request merged**